src/include/binder/copy/bound_copy_from.h

```diff
@@ -22,9 +22,9 @@ struct ExtraBoundCopyFromInfo {
 struct BoundCopyFromInfo {
     // Table entry to copy into.
     catalog::TableCatalogEntry* tableEntry;
-    // Data source
+    // Data source.
     std::unique_ptr<BoundBaseScanSource> source;
-    // Row offset of input data to generate internal ID.
+    // Row offset.
     std::shared_ptr<Expression> offset;
```

Comment:
Can you revisit if we still need this expression? Or if there is a way we can also get rid of this one?

Reply:
I'm inclined we still need one. Because the columns are generated by scan operator. And we don't want every scan operator to generate an offset expression by default.